From 52e672e0533d6fe8011601ae7c73cb097a55e2f7 Mon Sep 17 00:00:00 2001
From: Alexander Martinz <amartinz@shiftphones.com>
Date: Fri, 27 Aug 2021 13:37:54 +0200
Subject: [PATCH 3/3] init: reboot to recovery on panic by default to allow
 reading pstore

Note: bootloaders can still change the reboot target using cmdline.

Change-Id: I982ddb0b5720c660026e9b8b2ec4d0c3196b6442
Signed-off-by: Alexander Martinz <amartinz@shiftphones.com>
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 init/reboot_utils.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/reboot_utils.cpp b/init/reboot_utils.cpp
index 547b1869f3..dd0fbbfe81 100644
--- a/init/reboot_utils.cpp
+++ b/init/reboot_utils.cpp
@@ -37,7 +37,7 @@
 namespace android {
 namespace init {
 
-static std::string init_fatal_reboot_target = "bootloader";
+static std::string init_fatal_reboot_target = "recovery";
 static bool init_fatal_panic = false;
 
 // this needs to read the /proc/* files directly because it is called before
-- 
2.49.0

